calculators/candidate_supporting_expenditures.rb

```diff
@@ -5,12 +5,24 @@ def initialize(candidates: [], ballot_measures: [], committees: [])
   end
 
   def fetch
-    # Get the total indepedent expenditures for candidates by date.
+    # Get the total expenditures for candidates by committee.
+    # The Filer_NamL can be different in different records for the
+    # same Filer_ID.
     expenditures = ActiveRecord::Base.connection.execute(<<-SQL)
-      SELECT "Cand_ID", "Filer_ID", "Filer_NamL", SUM("Amount") as "Total"
-      FROM independent_candidate_expenditures
-      WHERE "Sup_Opp_Cd" = 'S'
-      GROUP BY "Cand_ID", "Filer_ID", "Filer_NamL";
+      SELECT "Cand_ID", i."Filer_ID", c."Filer_NamL", "Total"
+      FROM
+      (
+        SELECT "Cand_ID", "Filer_ID", SUM("Amount") as "Total"
+        FROM independent_candidate_expenditures
+        WHERE "Sup_Opp_Cd" = 'S'
+        GROUP BY "Cand_ID", "Filer_ID"
+      ) i
+      JOIN
+      (
+        SELECT DISTINCT ON ("Filer_ID") "Filer_ID", "Filer_NamL"
+        FROM committees
```

Comment:
Is the purpose of the join just to remove the candidates for which there is no committee, or is the `Filer_NamL` different in the committees sheet?

Reply:
Data in Contributions table can have different names for the committee.  The committee sheet can have multiple entries  for committees.